### ISSUE TITLE
Custom training goes wrong with "good hyperparameters"

### ISSUE THREAD

**Issue by xevolesi** [AlexeyAB/darknet#4751]
Hi @AlexeyAB !

I trained YOLOv3 with custom data set.
Classes:

- person;
- animal;
- car;
- truck;
- bus;
- bicycle;
- motorcycle;
- plate (car license plate).

I used YOLOv3 .cfg file with 416x416 input and added "good hyperparameters" from #4430 

Here is my chart.png

![chart](https://user-images.githubusercontent.com/17986847/73052277-cf77b700-3e95-11ea-9f02-ee859b8821b2.png)

Here is my yolov3.cfg: 
[yolov3-default-416-mosaic-scale-ciou.cfg.txt](https://github.com/AlexeyAB/darknet/files/4107236/yolov3-default-416-mosaic-scale-ciou.cfg.txt)

Some image examples from training and validation sets:
![Снимок экрана от 2020-01-24 10-47-00](https://user-images.githubusercontent.com/17986847/73052768-f84c7c00-3e96-11ea-940b-eb68ae472cbf.png)
![Снимок экрана от 2020-01-24 10-47-11](https://user-images.githubusercontent.com/17986847/73052771-fb476c80-3e96-11ea-95aa-b92f7861409d.png)

I checked my labels - it seems like it's correct, because earlier i trained default YOLOv3 and it gave me about 77% mAP.
Could you, please, tell me what could happen with my training?

Thanks!

**Comment by AlexeyAB** [comment]
Try to:

* remove `stopbackward=1`

* and add `max_delta=10` for each `[yolo]` layer

----

If it doesn't help try to

* or remove `mosaic=1`

* or use `iou_loss=mse` instead of `iou_loss=ciou`

**Comment by xevolesi** [comment]
Thanks!

I'll try both options and will notice you.

**Comment by xevolesi** [comment]
Hi, @AlexeyAB !

Your first suggestion worked well.

Here is final chart:

![Снимок экрана от 2020-01-31 12-42-45](https://user-images.githubusercontent.com/17986847/73529309-8ee4e400-4427-11ea-8b96-cd479d6d57ee.png)

Do you think this is the effect of max_delta or stopbackward removal?

**Comment by AlexeyAB** [comment]
> Do you think this is the effect of max_delta or stopbackward removal?

I don't know what did you do

**Comment by xevolesi** [comment]
Hi, @AlexeyAB !

> Try to:
> 
>     * remove `stopbackward=1`
> 
>     * and add `max_delta=10` for each `[yolo]` layer

i did this
